doc/modules/preprocessing.rst

```diff
@@ -423,3 +423,24 @@ values than observed values.
 
 :class:`Imputer` can be used in a Pipeline as a way to build a composite
 estimator that supports imputation. See :ref:`example_missing_values.py`
+
+
+Custom Transformers
+===================
+
+Often, you will want to convert an existing python function into a transformer
+to assist in data cleaning or processing. Users may implement a transformer from
```

Comment:
Reading this I might be confused. "Why will I want this? I can just do X = np.log(X) or something".
I think the most valuable use case is to grid search over the transformer function used. The paragraph might read better with a mention of this scenario.